projects/dotcms-field-elements/src/components/dot-form/dot-form-fields.tsx

```diff
@@ -0,0 +1,29 @@
+import { DotCMSContentTypeField } from '../../../../dotcms/src/lib/models';
```

Comment:
In the `tsconfig` of this project we can do this: https://github.com/dotCMS/core-web/blob/master/tsconfig.json#L23